mkdocs/tests/utils_tests.py

```diff
@@ -67,3 +67,33 @@ def test_create_media_urls(self):
         for path, expected_result in expected_results.items():
             urls = utils.create_media_urls(site_navigation, [path])
             self.assertEqual(urls[0], expected_result)
+
+    def test_yaml_load(self):
+        try:
+            from collections import OrderedDict
+        except ImportError:
+            # Don't test if can't import OrderdDict
+            # Exception can be removed when Py26 support is removed
+            return
+
+        yaml_text = '''
```

Comment:
To be consistent with the rest of mkdocs' tests, you might want to use [dedent][1] here. You can see many examples in [build_tests.py][2].

Otherwise, this looks great to me. Of course, approval is up to @d0ugal.

[1]: https://github.com/mkdocs/mkdocs/blob/master/mkdocs/tests/base.py#L7
[2]: https://github.com/mkdocs/mkdocs/blob/master/mkdocs/tests/build_tests.py#L42